src/Benchmarks/Startup.cs

```diff
@@ -84,22 +85,25 @@ public void ConfigureServices(IServiceCollection services)
                     break;
 
                 case DatabaseServer.MySql:
+                    services.AddEntityFrameworkMySql();
                     services.AddDbContextPool<ApplicationDbContext>(options => options.UseMySql(appSettings.ConnectionString));
 
                     if (Scenarios.Any("Raw") || Scenarios.Any("Dapper"))
                     {
                         services.AddSingleton<DbProviderFactory>(MySql.Data.MySqlClient.MySqlClientFactory.Instance);
                     }
                     break;
+
                 case DatabaseServer.MongoDb:
+                    // Reviewers: Is this actually needed for Mongo scenarios?
+                    services.AddEntityFrameworkSqlServer();
```

Comment:
I don't think so. I'd suggest to remove it. Worst case we'll see the mongodb benchmark fail and we can add it back.

Reply:
Definitively shouldn't be necessary for anything but the SQL Server case.